src/dbi.cpp

```diff
@@ -70,12 +70,12 @@ NAN_METHOD(DbiWrap::ctor) {
     bool needsTransaction = true;
     bool isOpen = false;
 
-    EnvWrap *ew = Nan::ObjectWrap::Unwrap<EnvWrap>(info[0]->ToObject());
+    EnvWrap *ew = Nan::ObjectWrap::Unwrap<EnvWrap>(Local<Object>::Cast(info[0]));
```

Comment:
I see a lot of changes from `ToSomething()` to `SomeThing::Cast` - looks good but is this still compatible with older node versions?

Reply:
So I checked on this and tested on Node V8.x. These `Cast` calls seemed to compile and work fine. The one issue I did run into is is String::Write is incompatible between v12 and v8, so I added a conditional directive based on node version to get it to compile on both v12 and v8: https://github.com/Venemo/node-lmdb/pull/150/files#diff-718077d5c48933b84cfa39dd863e1461R281
I couldn't get my build tools to work before version 8, not sure how far back we are trying to support.

Reply:
FWIW, according to https://nodejs.org/en/about/releases/, I think version 6 has reached end of life and version 8 is the oldest maintained node version, so v8 seems like a reasonable cutoff.